web_m2x_options/readme/USAGE.md

```diff
@@ -0,0 +1,106 @@
+## in the field's options dict
+
+`create` *boolean* (Default: depends if user have create rights)
+
+> Whether to display the "Create..." entry in dropdown panel.
+
+`create_edit` *boolean* (Default: depends if user have create rights)
+
+> Whether to display "Create and Edit..." entry in dropdown panel
+
+`m2o_dialog` *boolean* (Default: depends if user have create rights)
+
+> Whether to display the many2one dialog in case of validation error.
+
+`limit` *int* (Default: odoo default value is `8`)
+
+> Number of displayed record in drop-down panel
+
+`search_more` *boolean*
+
+> Used to force disable/enable search more button.
+
+`field_color` *string*
+
+> A string to define the field used to define color. This option has to
+> be used with colors.
+
+`colors` *dictionary*
+
+> A dictionary to link field value with a HTML color. This option has to
+> be used with field_color.
+
+`no_open_edit` *boolean* (Default: value of `no_open` which is `False`
+if not set)
+
+> Causes a many2one not to offer to click through in edit mode, but well
+> in read mode
+
+`open` *boolean* (Default: `False`)
+
+> Makes many2many_tags and one2many rows buttons that open the linked
+> resource
+
+`no_color_picker` *boolean* (Default: `False`)
```

Comment:
I couldn't make this option work. Are we sure it is working?

Reply:
There is no logic for this option in the v16 code. It is only referenced in the readme/translation files. Not sure what to do with this. Remove it from the readme or look at older versions of this module to see how it worked?